$NetBSD: patch-aa,v 1.3 2012/01/30 23:41:55 bubuchka Exp $

Compensate absense of long double functions in the NetBSD libm.

--- kccommon.h.orig	2011-04-30 15:52:25.000000000 +0000
+++ kccommon.h
@@ -68,7 +68,7 @@ extern "C" {
 #define snprintf  _snprintf
 #endif
 
-#if defined(__CYGWIN__)
+#if defined(__CYGWIN__) || defined(__NetBSD__)
 inline long double modfl(long double val, long double* iptr) {
   double integ;
   double fract = std::modf(val, &integ);
